packages/botonic-react/src/webchat/index.d.ts

```diff
@@ -1,7 +1,42 @@
+import * as core from '@botonic/core'
 import * as React from 'react'
 import { RefObject } from 'react'
 
-import { WebchatApp, WebchatArgs } from '../index'
+import { Reply, Webview } from '../components/index'
+import { Message, WebchatApp, WebchatArgs } from '../index'
+
+export interface WebchatState {
```

Comment:
I think `themeUpdates` is missing?

Reply:
yes, I added it and rebase, but I'm not sure. can you check? @asastre @vanbasten17  
```
export interface WebchatStateTheme {
  headerTitle: string
  brandColor: string
  brandImage: string
  triggerButtonImage: undefined
  textPlaceholder: string
  style: {
    fontFamily: string
  }
}

....
  theme: WebchatStateTheme
  themeUpdates: Partial<WebchatStateTheme>

```

Reply:
Yes, I can see it now